### PR TITLE
i18n: add update r1s r2s r4s cn translate

### PR DESCRIPTION
**Pull request by YuzukiTsuru**

### CODE REVIEW

File: ui/raidboss/data/07-dt/raid/r1s.ts [hunk 1]
```diff
@@ -411,12 +420,15 @@ const triggerSet: TriggerSet<Data> = {
         dirW: Outputs.dirW,
         insideOut: {
           en: 'Inside => Outside',
+          cn: '停 => 穿',
         },
         outsideIn: {
           en: 'Outside => Inside',
+          cn: '穿 => 穿',
         },
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r1s.ts:428]: This looks somehow puzzling.
Maybe `场内 => 场边` is better？

File: ui/raidboss/data/07-dt/raid/r1s.ts [hunk 2]
```diff
@@ -529,15 +542,19 @@ const triggerSet: TriggerSet<Data> = {
         dirW: Outputs.dirW,
         in: {
           en: 'In + Healer Stacks => Out',
+          cn: '停 + 治疗组分摊 => 穿',
         },
         out: {
           en: 'Out + Healer Stacks => In',
+          cn: '穿 + 治疗组分摊 => 停',
         },
```
Comment (Echoring) [inline ui/raidboss/data/07-dt/raid/r1s.ts:550]: Same as above.

File: ui/raidboss/data/07-dt/raid/r1s.ts (hunk 1 above)
Comment (YuzukiTsuru) [reply ui/raidboss/data/07-dt/raid/r1s.ts:428]: considering a method similar to p12s here, But it's better to do it this way as well

**Review by Echoring — APPROVED** [review]

File: ui/raidboss/data/07-dt/raid/r2s.ts [hunk 3]
```diff
@@ -74,12 +74,15 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         beatOne: {
           en: 'Soak towers - need 2-3 hearts',
+          cn: '踩塔 - 注意爱心层数',
```
Comment (MaikoTan) [inline ui/raidboss/data/07-dt/raid/r2s.ts:77]: Why not explicitly say hearts count here like English?

Comment (YuzukiTsuru) [reply ui/raidboss/data/07-dt/raid/r2s.ts:77]: If similar to English, there are the following translate:
- 踩塔 - 需要2-3层爱心: Very ambiguous which is like Only those with hearts can Soak on the tower.
- 踩塔 - 注意不要超过2-3层爱心: It sounds like don't go to Soak anymore if you got 2 or 3 hearts.
- 踩塔 - 注意不要超过3层爱心: The meaning is similar, but it's too long for tts

Perhaps there is a better way to express it, please suggest some for me.

Comment (Echoring) [reply ui/raidboss/data/07-dt/raid/r2s.ts:77]: How about `踩塔 - 踩到2~3颗心`？

**Review by MaikoTan — APPROVED** [review]

**Pull request merged**